src/odemis/driver/technolution.py

```diff
@@ -0,0 +1,544 @@
+    def __init__(self, name, role, parent, **kwargs):
+        super(MirrorDescanner, self).__init__(name, role, **kwargs)
+        self.parent = parent
+        # model.Emitter(name, role)
+        self.rotation = model.FloatContinuous(0, range=(0, 2 * numpy.pi), unit='rad, cw=+')
+        self.scanOffset = model.TupleContinuous((0.0, 0.0), range=((-10.0, -10.0), (10.0, 10.0)), unit='V,V')
+        self.scanGain = model.TupleContinuous((0.0, 0.0), range=((-10.0, -10.0), (10.0, 10.0)), unit='V,V')
+
+
+class MPPC(model.Detector):
+    def __init__(self, name, role, parent, **kwargs):
+        super(MPPC, self).__init__(name, role, **kwargs)
+        self.parent = parent
+
+        if not (hasattr(self.parent, "_MirrorDescanner")) or not (hasattr(self.parent, "_EBeamScanner")):
+            ConnectionError("Not all required children are defined (maybe definition order is not correct)")
+
+        self._field_data = FieldMetaData(None, None)  # Current field_data
+
+        # VA defauls values
+        self._shape = model.TupleVA((8, 8, 65536), unit="cell-images and depth")
+        self.path = model.StringVA('~/Pictures')
+        self.dataContent = model.StringVA('empty')
+        self.filename = model.StringVA(time.strftime("default-%H-%M-%S-%Y-%m-%d"))
+        self._server_URL = self.parent.server_URL
+        self.externalStorageURL_host = model.StringVA('host')
+        #NOTE: Do not write real username/password here since this is published on github in plaintext!
+        self.externalStorageURL_user = model.StringVA('user')
+        self.externalStorageURL_password = model.StringVA('password')
+        self.acqDelay = model.FloatVA(2.0, unit='s', setter=self._setAcqDelay)
+
+        # Cell acquisition parameters
+        #TODO K.K. Perform type check on intput into the VA's floats and ints
+        self.cellTranslation = model.ListVA([[[50, 50]] * self._shape.value[0]] * self._shape.value[1])
+        # TODO K.K. Put a check according to this input in the assemble_mega_field_check whih includes the counting up
+        #  per list
+        self.cellTranslation = model.ListVA([[[10 + j, 20 + j] for j in range(i,i+self._shape.value[0])]
+                                             for i in range(0,self._shape.value[1]*self._shape.value[0],self._shape.value[0])])
+        self.celldarkOffset = model.ListVA([[0] * self._shape.value[0]] * self._shape.value[1])
+        self.celldigitalGain = model.ListVA([[1.2] * self._shape.value[0]] * self._shape.value[1])
+        self.celldigitalGain = model.ListVA([[j for j in range(i, i+self._shape.value[0])]
+                                        for i in range(0,self._shape.value[1]*self._shape.value[0], self._shape.value[0])])
+        self.cellCompleteResolution = model.ResolutionVA((800, 800), ((10, 10), (900, 900)))
+
+        # Gather metadata from Siblings and own _meta_data
+        self.md_devices = [self._metadata, self.parent._MirrorDescanner._metadata, self.parent._EBeamScanner._metadata]
```

Comment:
I don't think the data is actually merged, it can still be changed later. This list should just containing reference to all the metadata. This allows to easily access the metadata of each device (MPPC, descanner, ebeam). Otherwise you need to create such a list later (in getMetadata for example). if you like to loop over the metadata and perform actions.
We could also define it there but I think it is nicer to create it in the _ _ init__

Reply:
Yes maybe you are right. Can you ask @pieleric about this? I am not sure what the elegant solution is.

Reply:
@pieleric Does this line just make a reference to the metadata of the devices? Meaning if metadata is updated in another component looking at this list you will see the update metdata? Or would this list become a copy of the metadat meaning you will not see update metadata?

Reply:
Typically, as _metadata is normally not overwritten in the components (as in `self._metadata = {...}`), this should work. You have references to dicts, which will be the correct ones for the whole life time of the components. Updating the dict (via `updateMetadata()`) will be reflected here. 

That said, you use this variable only once, in mergeMetadata(), so it'd be safer to just copy this line inside mergeMetadata, so that every time it's called, it gets *for sure* the right dict.